docs/docs/concepts/foundation/communication/public_private_calls/slow_updates_tree.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Privately access Historical Public Data
+---
+
+In Aztec, private and public execution environments are completely separate and operate with distinct state management. It is not possible for private functions to reliably access the most recent public data public state - only sequencers can do that.
```

Comment:
can you add a prerequisite to read the previous document first (public->private calls)